src/survey-score/survey-score.service.ts

```diff
@@ -4,13 +4,13 @@ import {
   NotFoundException,
 } from "@nestjs/common";
 import { ResponseTrackerStatusEnum, SurveyStatusEnum } from "@prisma/client";
```

Comment:
import properly it causes in test cases

Reply:
Test cases are in another PR changing here may cause merge conflicts